### PR TITLE
[FIX] invalidate_cache: avoid f-string in warning

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Fixes https://github.com/OCA/openupgradelib/pull/364.

Selection_3003

### CODE REVIEW

File: openupgradelib/openupgrade_tools.py [hunk 1]
```diff
@@ -368,9 +368,11 @@ def invalidate_cache(env, flush=True):
     # Warning on possibly untested versions where chunked might not work as expected
     if version > 17:  # unreleased version at this time
         logger.warning(
-            f"openupgradelib.invalidate_cache hasn't been tested on Odoo {version}. "
-            "Please report any issue you may find and consider bumping this warning "
-            "to the next version otherwise."
+            """openupgradelib.invalidate_cache hasn't been tested on Odoo {}.
```
Comment (pedrobaeza) [inline openupgradelib/openupgrade_tools.py:371]: Don't use triple quote, as that preserves the line feeds and indentations.

Comment (MiquelRForgeFlow) [reply openupgradelib/openupgrade_tools.py:371]: What should I use?

Comment (pedrobaeza) [reply openupgradelib/openupgrade_tools.py:371]: Simple quotes, as they were before.

Comment (MiquelRForgeFlow) [reply openupgradelib/openupgrade_tools.py]: Done

**Review by pedrobaeza — APPROVED** [review]

**Review by HviorForgeFlow — APPROVED** [review]

**Review by MateuGForgeFlow — APPROVED** [review]

**Review by RicardCForgeFlow — APPROVED** [review]

**Comment by ivantodorovich** [comment]
Thanks!

Sorry for this, I forgot about the old python versions we support here 😓

**Pull request merged**